### DISCUSSION TITLE
Version 24.3 release notes

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#103] [General]
Hello,

Do you have a release note of the new versions of BaramMesh and BaramFlow? 

Thank you?

Julien

**Comment by jiban** [comment]
Sure, you can see it [here](https://baramcfd.org/docs/release_notes/)

-- Jake

**Comment by torodecamargue** [comment]
Hello Jake,

I don't see the frozen flow feature. You said that it will be included in the next release right? 

The use of pressure jump BC in multi cores is corrected in 24.3 ?  

The display of Mesh is manual or automatic like in previous version ? 

I don't see radiation model P1 etc

Julien

**Comment by jiban** [comment]
Hi Julien,

1. For the frozen flow, you can control the equations in *Solution > Numerical Conditions > Advanced* menu.
    > * **Equation Control:** Equations can now be controlled to solve or not.

    ![image](https://github.com/user-attachments/assets/a631bb8f-69b4-495e-b467-62b1b4eb9240)

1. For **Porous Jump** BC in multi-core issue (it might be issue #75 ), I think it's fixed.
    > * **Boundary Type Compatibility:** Interface can be used as cyclic boundaries. The boundary pair of an interface now have same face ordering. The order was different in some cases before.

    You may generate the mesh with BaramMesh again and try it with BaramFlow.

1. The control of Mesh display in BaramMesh is under development and will be included in next release. ( v24.4.0 )

1. Unfortunately we don't have enough resource to develop radiation model for now.  
    Business team in my company schedules development resource according to business priority. As you may guess, customer-funding requests have priority. I'm sorry for that.

Best regards,
Jake

**Comment by MTODC** [comment]
Hello @jiban,
I tried the Porous Jump BC in multi-core but it didn't work, same behaviour of BaramFlow v. 24.2.1.

Best regards

**Reply by jiban** [reply]
Hello @MTODC,
Did you try it with the same mesh or case you had used before?
Please note that you should generated the mesh again with the latest BaramMesh, which is v24.3.1 at the moment.

Best regards,
Jake
